### PR TITLE
Update written content, add links, icons, tech-stacks

### PR DESCRIPTION
**Pull request by zantetsuken88**
Changes viewable [here](https://zantetsuken88.github.io/lee-stewart/)

- Major overhaul of written content which hadn't really been updated for quite a few years. Updates should include a more tangible sense of my personal achievements/impact in previous roles, more targeted references to particular tech.
- Added a couple of tooltips to elaborate certain terminology where inclusion in the main text felt excessive. 
- Added icons for each company that links to their website on click. 
- Added animated tech stack drawer where relevent

### CODE REVIEW

**Comment by zantetsuken88** [comment]
The large number of files changed is misleading! Many of them are images/svg icons which don't need to be code reviewed, plus config files etc.

File: src/components/RoleSection.jsx [hunk 1]
```diff
@@ -19,18 +50,31 @@ export default function RoleSection ({ data }) {
...
+            className={`chip chip${data.chipClass} chip${data.chipClass}-clickable`}
             variant='outlined'
-            label={<Typography variant='body2'>{data.company}</Typography>}/>
+            clickable
+            label={
+              <a className='company-label' href={data.website} target='_blank' rel='noopener noreferrer'>
+                <Typography variant='body2'>{data.company}</Typography>
+                <img src={link} alt='link icon'/>
+              </a>
+            }/>
```
Comment (julesnuggy) [inline src/components/RoleSection.jsx:62]: The `/>` should be on a new line

File: src/components/RoleSection.jsx [hunk 2]
```diff
@@ -19,18 +50,31 @@ export default function RoleSection ({ data }) {
...
           <Chip className={`chip chip${data.chipClass} chip${data.chipClass}-inner`}
             label={<Typography variant='body2'>{data.jobTitle}</Typography>}/>
           <Chip className={`chip chip${data.chipClass} chip${data.chipClass}-outer`}
             variant='outlined'
             label={<Typography variant='body2'>{data.yearRange}</Typography>}/>
```
Comment (julesnuggy) [inline src/components/RoleSection.jsx:67]: These 3 `/>` should be on a new line

File: src/components/RoleSection.jsx [hunk 3]
```diff
@@ -19,18 +50,31 @@ export default function RoleSection ({ data }) {
...
         </div>
-        {data.jobDescription.map((bullet, i) =>
-          <Typography variant='body2' key={`data.jobTitle${i}`}>{bullet}</Typography>
-        )}
+        <ul>
+          {jobDesc.map((bullet, i) => {
+            return <Typography className='role-text' variant='body2' component='li' key={`${data.jobTitle}${i}`}>
+              { bullet }
+            </Typography>;
```
Comment (julesnuggy) [inline src/components/RoleSection.jsx:73]: An immediate return would be neater :-)

File: src/components/RoleSection.jsx [hunk 4]
```diff
@@ -19,18 +50,31 @@ export default function RoleSection ({ data }) {
...
+          }
+          )}
```
Comment (julesnuggy) [inline src/components/RoleSection.jsx:75]: This should be grouped with previous `}`

File: src/components/TechStack.jsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+  const buttonSelected = iconClass === iconClasses.showIcons ? 'stack-button-selected' : '';
+
+  const Icons = iconMapping[techStack];
+
+  return (
+    <div className='tech-stack-container'>
+      <button className={`stack-button ${buttonSelected}`} onClick={() => handleClick()}><Typography variant='caption'>Tech Stack</Typography></button>
+      <div className='stack-container'>
+        { <Icons iconClass={iconClass}/> }
```
Comment (julesnuggy) [inline src/components/TechStack.jsx:33]: I don't think these `{ }` are necessary?

File: src/components/TechStack.jsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    iconClass === showIcons ? setIconClass(hideIcons) : setIconClass(showIcons);
+  };
+
+  const buttonSelected = iconClass === iconClasses.showIcons ? 'stack-button-selected' : '';
+
+  const Icons = iconMapping[techStack];
+
+  return (
+    <div className='tech-stack-container'>
+      <button className={`stack-button ${buttonSelected}`} onClick={() => handleClick()}><Typography variant='caption'>Tech Stack</Typography></button>
```
Comment (julesnuggy) [inline src/components/TechStack.jsx:31]: This is a very long line - would be better to structure it out.

File: src/media/techicons/TechIcons.jsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+// Starling Logos
```
Comment (julesnuggy) [inline src/media/techicons/TechIcons.jsx:2]: I think this comment is redundant now?

**Review by julesnuggy — CHANGES_REQUESTED** [review]
Looks good - just a few minor points mostly about linting.

It would be good if you could also:
1) Include a link to your actual CV website in the README and probably also your Github profile - it's very hard to search for / find it at the moment. In fact, the README probably shouldn't have any of the React auto-generated text.

2) The company link Chip components are a bit confusing. They look and behave like buttons but you have to click on the text to be taken to the company website. It would be more intuitive / better UI/UX if clicking just the pill itself took you to the website.

**Pull request merged**